### PR TITLE
DBZ-3897 Upgrade Antora to 3.0.0-alpha.9

### PR DESCRIPTION
**Pull request by uidoyen**

### CODE REVIEW

File: website-builder/Dockerfile [hunk 1]
```diff
@@ -32,6 +29,7 @@ ADD Gemfile.lock Gemfile.lock
 RUN bundle install 
 
 WORKDIR $SITE_HOME
+VOLUME [ $SITE_HOME ]
```
Comment (gunnarmorling) [inline website-builder/Dockerfile:32]: @uidoyen, do we truly need this? I just had removed it (see https://github.com/debezium/docker-images/commit/00a1d9b482b1f6b55edd44b0b43edbaba85b144f), as it gave me problems with recent Docker versions. And it shouldn't be part of the Dockerfile itself to begin with, but can be specified upon docker run... . WDYT?

Comment (uidoyen) [reply website-builder/Dockerfile:32]: Actually, I have added it test locally. I am removing it again. Thanks!

Comment (gunnarmorling) [reply website-builder/Dockerfile]: Excellent, thx!

**Comment by gunnarmorling** [comment]
Applied; thx, @uidoyen!

**Pull request merged**